test/test_utils/ecs.py

```diff
@@ -27,7 +27,7 @@
 
 ECS_INSTANCE_ROLE_ARN = "arn:aws:iam::669063966089:instance-profile/ecsInstanceRole"
 ECS_INSTANCE_ROLE_NAME = "ecsInstanceRole"
-TENSORFLOW_MODELS_BUCKET = "s3://tensoflow-trained-models"
+TENSORFLOW_MODELS_BUCKET = "s3://aws-tensorflow-dlc-models"
```

Comment:
This models bucket may be used for other models as well -- did you sync everything from the original bucket? if not, you could get some unexpected failures